docs/search_relevance/tests/test-6-english-tokeniser-and-contributors.md

```diff
@@ -0,0 +1,31 @@
+# Test 6 - English tokeniser and Contributors
+
+## Candidates
+
+A variant of the scoring tiers query introduced in Test 5 was tested. The variant adds an english analyser \(to capture peculiarities of apostrophes in queries like `gray's anatomy`\), and includes contributors in the list of queried fields.
+
+![rough form of a scoring tiers query](../../.gitbook/assets/untitled_drawing.png)
+
+## Results
+
+The test ran for three weeks over the Christmas break. It's worth bearing in mind that the audience we're serving is likely to be less discerning and research-focused if our researcher audience are also less active over these weeks, but the size of the dataset \(~9,000 sessions, 100,000 events\) should still give us reliable indication of goodness.
+
+**Click through rate**
+
+|  | default scoring tiers query | variant |
+| :--- | :--- | :--- |
+| first page only | 0.238 | 0.220 |
+| beyond first page | 0.564 | 0.533 |
+
+**Click distribution**
+
+![download-1](https://user-images.githubusercontent.com/11006680/71817901-408a3280-307f-11ea-95ee-f8aaea9e2f56.png)
+
+## Conclusions
+
+The data indicates that the new variant query performs worse than the default, particularly at the top end of the list of results \(where worse performance is indicated by fewer clicks\). 
+
+The difference isn't substantial, and both queries seem to perform better than some of our previous candidates. However, we should still step back and evaluate why performance is lower before proposing a new set of changes.
```

Comment:
I don't think we should suggest rolling back, but things we can do moving forward.

Maybe removing the english analyser as this is the hunch. 

Thinking of this, while adding features e.g. adding contributors, is there an amount of negative vatriation we will permit in the name of adding something we want?

Another option is saying:
> Intention: Find things made by a contributor

And then work on that until it's better? I like this more and should maybe be our next step.

I imagine the workflow to be:
- `ScoringTiers` as `A`
- `ScoringTiersWithContributorsV1` as `B`
- Keep updating `ScoringTiersWithContributorsVn` until it beats `A`
- Make `ScoringTiersWithContributorsVn` default
- Create new intention, iterate on that

How does that sound? I can capture this and share it with the team if we think it's good?

Reply:
that sounds very sensible to me - might be worth adding to the query design page in the docs?